gdist.py

```diff
@@ -0,0 +1,76 @@
+from ctypes import POINTER, CDLL, c_int, c_uint, c_double
+import glob
+
+import numpy as np
+import scipy
+
+libfile = glob.glob('build/*/gdist*.so')[0]
+
+lib = CDLL(libfile)
+
+
+class Gdist(object):
+    def __init__(self):
+        lib.compute_gdist.argtypes = [
+            c_uint,
+            c_uint,
+            np.ctypeslib.ndpointer(dtype=np.float64),
+            np.ctypeslib.ndpointer(dtype=np.int32),
+            c_uint,
+            c_uint,
+            np.ctypeslib.ndpointer(dtype=np.int32),
+            np.ctypeslib.ndpointer(dtype=np.int32),
+            c_double,
+        ]
+        lib.compute_gdist.restype = POINTER(c_double)
+
+    def compute_gdist(
+        self,
+        number_of_vertices,
+        number_of_triangles,
+        vertices,
+        triangles,
+        number_of_source_indices,
+        number_of_target_indices,
+        source_indices_array,
+        target_indices_array,
+        distance_limit
+    ):
+        return lib.compute_gdist(
+            number_of_vertices,
+            number_of_triangles,
+            vertices,
+            triangles,
+            number_of_source_indices,
+            number_of_target_indices,
+            source_indices_array,
+            target_indices_array,
+            distance_limit
+        )
+
+
+def compute_gdist(
+    vertices,
+    triangles,
+    source_indices=None,
+    target_indices=None,
+    max_distance=10**100,
```

Comment:
Please check the current default, which is 40.0 IIRC

Reply:
Current default value is:
https://github.com/the-virtual-brain/tvb-gdist/blob/af53f955616d43a71d4b81bce41bfef08d08e25a/gdist.pyx#L119

which in turn is:
https://github.com/the-virtual-brain/tvb-gdist/blob/c44c167c4eb8955cf9d20da1507f6cdf56e2bfc3/geodesic_library/geodesic_constants_and_simple_functions.h#L20